test/gst-va/encode/10bit/vp9.py

```diff
@@ -0,0 +1,94 @@
+###
+### Copyright (C) 2023 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from .....lib import *
+from .....lib.gstreamer.va.util import *
+from .....lib.gstreamer.va.encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "10bit")
+
+@slash.requires(*have_gst_element("vavp9lpenc"))
```

Comment:
This should be at the `VP9_10EncoderLPTest` subclass

Reply:
updated